packages/blade/src/components/Form/SelectorLabel.d.ts

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export * from './SelectorLabel.web';
+
+export type SelectorLabelProps = {
+  children: React.ReactNode;
+  /**
+   * Pass only on react-native
+   */
+  inputProps: any; // Partial<Extract<InputProps, { onPress: any }>>;
```

Comment:
any?? anyyyyy?? anyyyyyyyyyyy by @anuraghazra?😮

Reply:
Sacrilegious!

Reply:
:p well unfortunately this InputProps can take either Radio or Checkbox thus for the time being I've made it any. If I feel we can later make it more strict after Radio is merged I'll do it.